### PR TITLE
[docs] add p-norm example

### PR DESCRIPTION
**Pull request by odow**
Came up on Discourse: https://discourse.julialang.org/t/nonlinear-optimization-using-jump-the-solver-does-not-support-nonlinear-problems-i-e-nlobjective-and-nlconstraint/95978

Preview: https://jump.dev/JuMP.jl/previews/PR3282/tutorials/conic/tips_and_tricks/#P-Norm

### CODE REVIEW

File: docs/src/tutorials/conic/tips_and_tricks.jl [hunk 1]
```diff
@@ -217,6 +217,28 @@ value(t), value(x)
 # that offers an alternative formulation that can be more efficient for some
 # formulations.
 
+# ## P-Norm
+
+# The p-norm ``||x||_p = \left(\sum\limits_{i} |x_i|^p\right)^{\frac{1}{p})``
```
Comment (odow) [inline docs/src/tutorials/conic/tips_and_tricks.jl:222]:
```suggestion
# The p-norm ``||x||_p = \left(\sum\limits_{i} |x_i|^p\right)^{\frac{1}{p}}``
```

**Comment by odow** [comment]
This probably comes up often enough that I wonder if it's worth a dedicated set and bridge?

File: docs/src/tutorials/conic/tips_and_tricks.jl [hunk 2]
```diff
@@ -217,6 +217,28 @@ value(t), value(x)
 # that offers an alternative formulation that can be more efficient for some
 # formulations.
 
+# ## P-Norm
+
+# The p-norm ``||x||_p = \left(\sum\limits_{i} |x_i|^p\right)^{\frac{1}{p}}``
+# can be modeled using a set of power cones. See the [Mosek Modeling Cookbook](https://docs.mosek.com/modeling-cookbook/powo.html#p-norm-cones)
```
Comment (blegat) [inline docs/src/tutorials/conic/tips_and_tricks.jl:223]: What do you mean by "set of power cones"

**Comment by blegat** [comment]
> This probably comes up often enough that I wonder if it's worth a dedicated set and bridge?

Yes, @chriscoey do you have something like that in Hypatia ?

**Comment by odow** [comment]
I've opened an issue: https://github.com/jump-dev/MathOptInterface.jl/issues/2118.

This doc PR can be merged without waiting for the resolution.

**Comment by chriscoey** [comment]
> > This probably comes up often enough that I wonder if it's worth a dedicated set and bridge?
> 
> Yes, @chriscoey do you have something like that in Hypatia ?

No, we didn't know a nice barrier for p-norm so we didn't try implementing anything.

File: docs/src/tutorials/conic/tips_and_tricks.jl (hunk 2 above)
Comment (odow) [inline docs/src/tutorials/conic/tips_and_tricks.jl:223]:
```suggestion
# can be modeled using [`MOI.PowerCone`](@ref)s. See the [Mosek Modeling Cookbook](https://docs.mosek.com/modeling-cookbook/powo.html#p-norm-cones)
```

**Pull request merged**